plugins/GSdx/GSSetting.cpp

```diff
@@ -123,11 +123,12 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "Makes textures partially or fully transparent as required by emulation. May cause unusual slowdowns for some games.";
 		case IDC_LOGZ:
 			return "Treat depth as logarithmic instead of linear. Recommended setting is on unless it causes graphical glitches.";
+		// Exclusive for Hardware Renderer	
+		case IDC_PRELOAD_GS:
+			return "Uploads GS data when rendering a new frame to reproduce some effects accurately. Fixes black screen issues in games like Primal.";
```

Comment:
> Wasn't it supposed to say Armored Core: Last Raven? Oh, also do a trailing whitespace check.

yeah, you're right. no clue how that got swapped up @_@ 

> ..It must seem like I'm just messing around with you. I assure you I am not.

I'm not sure what gave you that impression ! 

either way, fixed the trailing whitespace and ``IDC_PRELOAD_GS`` tooltip modification.